src/compass/utils/lut.py

```diff
@@ -80,8 +80,9 @@ def cumulative_correction_luts(burst, dem_path, tec_path,
         rg_lut_data += dry_los_tropo
 
     # Invert signs to correct for convention
-    # TO DO: add azimuth SET to LUT
-    az_lut_data = -(bistatic_delay.data + az_fm_mismatch.data)
+    #az_lut_data = -(bistatic_delay.data + az_fm_mismatch.data)
+    az_lut_data = -(bistatic_delay.data)
```

Comment:
```suggestion
    az_lut_data = -bistatic_delay.data
```
Parenthesis are redundant